src/Features/CSharp/Portable/AddImport/CSharpAddImportCodeFixProvider.cs

```diff
@@ -166,6 +167,7 @@ public CSharpAddImportCodeFixProvider()
         }
 
         /// <summary>For testing purposes only (so that tests can pass in mock values)</summary> 
+        [SuppressMessage("RoslynDiagnosticsReliability", "RS0034:Exported parts should have [ImportingConstructor]", Justification = "Used incorrectly by tests")]
```

Comment:
this check seems busted.  it says `Exported parts should have [ImportingConstructor]`.  But it does have one of those.

Reply:
That's the title of the analyzer. The message is:

> Exported parts should have a single [ImportingConstructor] of the correct form.